src/graph/entities/qg_edge.c

```diff
@@ -70,12 +70,19 @@ QGEdge *QGEdge_Clone
 	return e;
 }
 
+// returns true if edge represents a variable length path
+// e.g. ()-[*]->()
+// note: a zero length edge: ()-[*0]->() is considered a variable length edge
```

Comment:
why?

Reply:
We would like to treat zero length edges as variable length edges, this way our var-len traversal will be used and only the source node will be matched.